tools/shared.py

```diff
@@ -1650,9 +1650,9 @@ class ret:
           ret.undefs.append(symbol)
         elif status == 'C':
           ret.commons.append(symbol)
-        elif status == status.upper(): # all other uppercase statuses ('T', etc.) are normally defined symbols
```

Comment:
In http://llvm.org/docs/CommandGuide/llvm-nm.html, 'W', 'T', and 'D' are only uppercase literals other than 'U' and 'C' already handled above. Do you still think tests are necessary? Then which tests do I need to run?
Or if you prefer the code to be remained in the older way (when include_internal == False), I can do that too.